### ISSUE TITLE
Implement Age 3 cards for Unseen Expansion

### ISSUE THREAD

**Issue by ultimatefiend** [micahstairs/bga-innovation#1076]
This issue will track the implementation of all Age 3 cards for the Unseen expansion. Please update this comment before working on any of these cards (if the work is unblocked) so that we don't have duplicate work being done. We no longer need to file a separate issue for each card, since a consolidated list of the cards will keep us more organized.

- [x] 505 : Brethren of Purity
- [x] 506 : Secretum Secretorum
BLOCKED : #1107
- [x] 507 : Knights Templar
- [x] 508 : Red Envelope
- [x] 509 : Cliffhanger
BLOCKED : #1108
- [x] 510 : Smuggling
- [x] 511 : Freemasons
- [x] 512 : Secret Police
- [x] 513 : Masquerade
- [x] 514 : Taqiyya
